notebooks/community/model_garden/model_garden_pytorch_llama2_peft.ipynb

```diff
@@ -0,0 +1,756 @@
+{
```

Comment:
<p>serving in Vertex -&gt; serving in Vertex AI</p><p> </p><p>gcs -&gt; Cloud Storage</p><p> </p><p>huggingface -&gt; Hugging Face</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>done</p>